windows-pull-run.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+
+if not "%1"=="" (
+    if not "%1"=="-o" (
+           echo script usage: windows-pull-run.bat [-o]
+           echo -o: disables automatic opening of the browser
+           exit /b 1
+       )
```

Comment:
```suggestion
        echo script usage: windows-pull-run.bat [-o]
        echo     -o: disables automatic opening of the browser
        exit /b 1
    )
```

Reply:
fixed!

Reply:
Looks like the indentation of those lines still has to be decreased by 3 spaces in the file